apps/src/code-studio/components/ImagePicker.jsx

```diff
@@ -26,40 +27,48 @@ export default class ImagePicker extends React.Component {
     useFilesApi: PropTypes.bool,
     soundPlayer: PropTypes.object,
     disableAudioRecording: PropTypes.bool,
+    currentValue: PropTypes.string,
+    currentImageType: PropTypes.string,
     //For logging purposes
     projectId: PropTypes.string,
     elementId: PropTypes.string
   };
 
-  state = {mode: 'files'};
+  state = {mode: ImageMode.FILE};
 
   getAssetNameWithPrefix = icon => {
     this.props.assetChosen(ICON_PREFIX + icon);
   };
 
-  setIconMode = () => this.setState({mode: 'icons'});
-
-  setFileMode = () => this.setState({mode: 'files'});
+  setMode = mode => {
+    this.setState({mode: mode});
```

Comment:
[super nit, and more of an FYI] if the key and value are the same, you can shorthand like this:
```javascript
this.setState({mode});
```